config/services.php

```diff
@@ -6,5 +6,9 @@
     'factories' => [
         Plugin\Guard\RouteGuard::class => Container\Plugin\Guard\RouteGuardFactory::class,
         Plugin\Guard\FinalizeGuard::class => Container\Plugin\Guard\FinalizeGuardFactory::class,
+        // static factory calls with individual config id
+        'special_command_bus' => [\Prooph\ServiceBus\Container\CommandBusFactory::class, 'special'],
```

Comment:
You're right, thanks for clarification. PR is good for me.
Am 06.03.2016 22:02 schrieb "Sandro Keil" <notifications@github.com>:

> In config/services.php
> <https://github.com/prooph/service-bus/pull/116#discussion_r55149377>:
>
> > @@ -6,5 +6,9 @@
> >      'factories' => [
> >          Plugin\Guard\RouteGuard::class => Container\Plugin\Guard\RouteGuardFactory::class,
> >          Plugin\Guard\FinalizeGuard::class => Container\Plugin\Guard\FinalizeGuardFactory::class,
> > +        // static factory calls with individual config id
> > +        'special_command_bus' => [\Prooph\ServiceBus\Container\CommandBusFactory::class, 'special'],
>
> Is this file used anywhere? I thought it was only an example file, see
> docs.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prooph/service-bus/pull/116/files#r55149377>.
>